test/xds/xds_security_config_nack_test.go

```diff
@@ -144,8 +147,22 @@ func (s) TestUnmarshalListener_WithUpdateValidatorFunc(t *testing.T) {
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			managementServer, nodeID, bootstrapContents, resolver, cleanup1 := e2e.SetupManagementServer(t, e2e.ManagementServerOptions{})
-			defer cleanup1()
+			managementServer := e2e.StartManagementServer(t, e2e.ManagementServerOptions{AllowResourceSubset: true})
```

Comment:
why didnt we use `setupManagementServerAndResolver` here?

Reply:
Missed it. Done now.